addons/web/static/src/js/views/search_panel_model_extension.js

```diff
@@ -0,0 +1,616 @@
+                    );
+                } else {
+                    section.domain = attrs.domain || "[]";
+                    section.groupBy = attrs.groupby;
+                    section.icon = section.icon || "fa-filter";
+                }
+                this.state.sections.set(section.id, section);
+            });
+        }
+
+        /**co
+         * Fetch values for each category at startup. At reload a category is only
+         * fetched if the searchDomain changes and displayCounters is true for it.
+         * @private
+         * @param {Object} options
+         * @param {boolean} options.searchDomainHasChanged
+         * @param {boolean} [options.initialLoad=false]
+         * @returns {Promise} resolved when all categories have been fetched
+         */
+        _fetchCategories(options) {
+            const proms = [];
+            const { initialLoad, searchDomainHasChanged } = options;
+            for (const category of this.categories) {
+                const { enableCounters, expand, hierarchize, limit } = category;
+                if (initialLoad || enableCounters || (searchDomainHasChanged && !expand)) {
+                    const prom = this.env.services
+                        .rpc({
+                            method: "search_panel_select_range",
+                            model: this.config.modelName,
+                            args: [category.fieldName],
+                            kwargs: {
+                                category_domain: this._getCategoryDomain(
+                                    category.id
+                                ),
+                                enable_counters: enableCounters,
+                                expand,
+                                filter_domain: this._getFilterDomain(),
+                                hierarchize,
+                                limit,
+                                search_domain: this.searchDomain,
+                            },
+                        })
+                        .then(
+                            (result) => this._createCategoryTree(category.id, result)
+                        );
+                    proms.push(prom);
+                }
+            }
+            return Promise.all(proms);
+        }
+
+        /**
+         * Fetch values for each filter. This is done at startup, and at each reload
+         * (when the controlPanel or searchPanel domain changes).
+         * @private
+         * @param {Object} options
+         * @param {boolean} options.searchDomainHasChanged
+         * @param {boolean} [options.initialLoad=false]
+         * @returns {Promise} resolved when all filters have been fetched
+         */
+        _fetchFilters(options) {
+            const evalContext = {};
+            for (const category of this.categories) {
+                // It is weird to have to put a domain of the form [categ_field, "=", false] or the like in case activeValueId is __all__!
```

Comment:
It is weird to have to put a domain of the form [categ_field, "=", false] or the like in case activeValueId is __all__!
I think it was done that way because of documents specs
But i don't have a better idea --> remove parts of domain that uses a category with __all__ as active value? --> too difficult i think

Reply:
still valid

Reply:
Done